cmd/notation/plugin.go

```diff
@@ -62,3 +108,169 @@ func listPlugins(command *cobra.Command) error {
 	}
 	return tw.Flush()
 }
+
+func installPlugin(command *cobra.Command, args []string, force bool) error {
+	if len(args) != 1 {
+		return errors.New("missing plugin package")
+	}
+
+	pluginPath := args[0]
+
+	var pluginBinary string
+	// if plugin contains a file path split and select the last element in the array
+	if strings.Contains(pluginPath, "/") {
+		pluginBinary = strings.Split(pluginPath, "/")[len(strings.Split(pluginPath, "/"))-1]
+	} else {
+		pluginBinary = pluginPath
+	}
+
+	// get plugin metadata
+	cmd := exec.Command("./"+pluginPath, "get-plugin-metadata")
+
+	output, err := cmd.Output()
+	if err != nil {
+		panic(err)
+	}
+
+	var newPlugin map[string]interface{}
+	err = json.Unmarshal([]byte(output), &newPlugin)
+	if err != nil {
+		return err
+	}
+
+	pluginName := newPlugin["name"].(string)
+	newPluginVersion := newPlugin["version"].(string)
```

Comment:
As it's type assertion on map values, we need:
```suggestion
	pluginName, ok := newPlugin["name"].(string)
	if !ok {
	    xxx
	}
	newPluginVersion, ok := newPlugin["version"].(string)
	if !ok {
	    xxx
	}
```